conf/messages

```diff
@@ -20,12 +20,16 @@ transferAgreement.english=The records are all in English
 transferAgreement.digital=The records are all Digital
 transferAgreement.droAppraisalSelection=I confirm that the DRO has signed off on the appraisal and selection decision.
 transferAgreement.droSensitivity=I confirm that the DRO has signed off on the sensitivity review and that all records are OPEN i.e. no FOI exemptions apply to these records.
-upload.header=Upload
-upload.title=Upload
+upload.header=Upload Records
+upload.title=Upload Records
+upload.message=Select a folder
+upload.details=Which file extensions are not permitted?
+upload.summary=Cannot upload the following file extensions: .zip, thumbs.db, .exe
 records.header=Records
 records.title=Records
 
 # Progress indicator step numbers
 seriesDetails.progress=1
 transferAgreement.progress=2
-progress.totalSteps=2
\ No newline at end of file
+progress.uploadStep=3
```

Comment:
Just a rewording change: I think this should be `upload.progress` to be consistent with the other steps.